File: src/main/java/no/nav/syfo/rest/domain/RSStilling.java
package no.nav.syfo.rest.domain;

import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.experimental.Accessors;

import java.math.BigDecimal;
import java.time.LocalDate;


@Data
@Accessors(fluent = true, chain = true)
@EqualsAndHashCode
public class RSStilling {
    public String virksomhetsnummer;
    public String yrke;
    public BigDecimal prosent;
    public LocalDate fom;
    public LocalDate tom;

}

File: src/main/kotlin/no/nav/syfo/arbeidsforhold/controller/RSStilling.kt
package no.nav.syfo.arbeidsforhold.controller

import java.math.BigDecimal
import java.time.LocalDate

data class RSStilling(
    val virksomhetsnummer: String? = null,
    val yrke: String? = null,
    val prosent: BigDecimal? = null,
    val fom: LocalDate? = null,
    val tom: LocalDate? = null
)
